### PR TITLE
Added docs controller to expose Swagger API docs

### PR DESCRIPTION
**Pull request by ianlovell**

### CODE REVIEW

**Comment by csscottlogic** [comment]
:shipit:

File: Speakers/api.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    "type": "string",
+                    "description": "Surname."
+                },
+                "rating": {
+                    "type": "integer",
+                    "description": "Rating (from 1 to 5)"
+                },
+                "imageUrl": {
+                    "type": "string",
+                    "description": "Image URL representing the product."
```
Comment (lclarksl) [inline Speakers/api.json]: Product?

File: Speakers/api.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    "host": "api.bris.tech",
+    "schemes": [
+        "https"
+    ],
+    "basePath": "/",
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/speakers": {
```
Comment (lclarksl) [inline Speakers/api.json]: Should be /talkoutlines now

**Comment by lclarksl** [comment]
:shipit: other than minor points

**Comment by ianlovell** [comment]
I will merge, as the api.json will change next commit anyway. As for /talkoutlines, this will change to /talks, unless there's a reason not to consider that entity a talk.

**Pull request merged**